recommendations/sandbox/keda.md

```diff
@@ -0,0 +1,203 @@
+
+The Kubernetes Event-driven Autoscaling (KEDA) project does not want to reinvent the wheel but build on standards instead and is complimentary to Kubernetes. Next to that, we have support for CNCF projects like Prometheus and NATS by providing scalers for them as well and package our product as a Helm chart (2.x & 3.x) which is available on Helm Hub. Lastly, we are vendor-neutral by supporting all major clouds and open-source technologies like Redis.
+
+While the scaling features of Kubernetes Event-driven Autoscaling (KEDA) are important, we are a strong believer of making the product itself operable as well. By using Operator SDK we also want to allow operators to efficiently manage the infrastructure necessary to run Kubernetes Event-driven Autoscaling (KEDA) and are planning to provide a better operational experience as a whole by providing a CLI, dashboard, etc.
+
+Security is one of our main focuses and we will keep on investing in that - This is why pod identity has been one of our main focuses for 1.0 and will continue to support more identity providers over time.
+
+## Value-add to the CNCF ecosystem
+
+The Kubernetes Event-driven Autoscaling (KEDA) project does not want to reinvent the wheel but build on standards instead and is complimentary to Kubernetes.
+
+From the beginning intent was open and community driven (open community standups, MIT License, non-Microsoft branded). 
+
+CNCF provides a vendor neutral home for key serverless capability.
+
+
+## Alignment with other CNCF projects
+
+KEDA augments and enhances many CNCF projects by adding event driven scale, including:
+
+- Kubernetes itself (helping drive features back into things like the HPA / Cluster autoscaler)
+- Virtual Kubelet
+- NATS
+- Promethues
+- Strimzi
+- Helm
+
+## Anticipated use cases
+
+- Serverless container and FaaS scaling
+- Data processing pipelines
+- Application scaling
+- Batch processing
+
+## Alignment with SIG Reference Model
+
+SIG-Runtime
+
+## High level architecture
+
+KEDA performs two key roles within Kubernetes. First, it acts as an agent to activate and deactivate a deployment to scale to and from zero on no events. This is one of the primary roles of the keda-operator container that runs when you install KEDA.
+
+Second, KEDA acts as a Kubernetes metrics server to expose rich event data like queue length or stream lag to the horizontal pod autoscaler to drive scale out. It is up to the deployment to then consume the events directly from the source. This preserves rich event integration and enables gestures like completing or abandoning queue messages to work out of the box. The metric serving is the primary role of the keda-operator-metrics-apiserver container that runs when you install KEDA.
+
+![KEDA Architecture](https://i.imgur.com/lBVkWsH.png)
+
+# Formal Requirements
+
+Sponsors: TBD
+
+# CNCF IP Policy
+
+	cloud.google.com/go (Apache License 2.0)
```

Comment:
Nitpick, indentation for all these. Can be either:

- cloud.google.com/go (Apache License 2.0)
...
or 

cloud.google.com/go (Apache License 2.0)
...

Reply:
Done